api4/license.go

```diff
@@ -305,6 +302,29 @@ func getPrevTrialLicense(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(model.MapToJSON(clientLicense)))
 }
 
+func getOrCreateTrueUpReviewStatus(c *Context) (*model.TrueUpReviewStatus, bool) {
+	nextDueDate := utils.GetNextTrueUpReviewDueDate(time.Now())
+	status, err := c.App.Srv().Store().TrueUpReview().GetTrueUpReviewStatus(nextDueDate.UnixMilli())
+	if err != nil {
+		var nfErr *store.ErrNotFound
+		switch {
+		case errors.As(err, &nfErr):
+			c.Logger.Warn("Could not find true up review status")
```

Comment:
Thank you for using the context logger :)

Reply:
Credits to @ConorMacpherson !